test/features/caseView.feature

```diff
@@ -1,7 +1,21 @@
-@caseView
+@functional @caseView
 Feature: Set of scenarios testing case view functionality
 
   Background:
+    Given the following definition for 'tab case fields'
+      | ID               | Label             | Tab Display order   | CaseFieldID          | Field Display order | Tab show condition               |
+      | History          | History           | 1                   | CaseHistory          | 1                   |                                  |
+      | Tab2             | Tab2              | 3                   | TextField4           | 3                   |                                  |
+      | Tab2             | Tab2              | 3                   | TextField3           | 2                   |                                  |
+      | Tab2             | Tab2              | 3                   | TextField2           | 1                   |                                  |
+      | Tab1             | Tab1              | 2                   | TextField1           | 1                   |                                  |
+      | Tab3             | Tab3              | 4                   | ConditionalText      | 3                   |                                  |
+      | Tab3             | Tab3              | 4                   | ConditionalFixedList | 3                   |                                  |
+      | Tab3             | Tab3              | 4                   | ConditionalYesNo     | 3                   |                                  |
+      | ConditionalTab1  | Conditional Tab 1 | 5                   | TextField5           | 1                   | ConditionalText="showmethemoney" |
+      | ConditionalTab2  | Conditional Tab 2 | 6                   | TextField6           | 1                   | ConditionalFixedList="TRUE"      |
+      | ConditionalTab3  | Conditional Tab 3 | 7                   | TextField7           | 1                   | ConditionalYesNo="Yes"           |
+
     Given I have logged in
```

Comment:
I can see in the source file a step called:
`Given a case containing Tabs functionality exists`
I was thinking, is this step still needed now that we have the new step `Given the following definition for 'tab case fields'`? Aren't both steps verifying the same thing?